src/UI/Conversation.jsx

```diff
@@ -289,9 +290,29 @@ class Conversation extends Component {
 		this.props.api.updateMessagesStatus(STATUS_SEEN, messageIds);
 	};
 
+	shouldRenderWelcomeMessageUnderDate = (message) => {
+		const msInSecond = 1000;
+		const isMessageTimeToday
+			= this.getDateFromTimestamp(message.time) === this.getDateFromTimestamp(Date.now() / msInSecond);
+		const isDateRendered = this.renderedDates.includes(this.getDateFromTimestamp(message.time));
```

Comment:
Kan het voorkomen dat dit false is dan?

Voor deze aanroep zit toch altijd de `this.setRenderedDate(this.getDateFromTimestamp(message.time))` ?

Reply:
ik had deze erin zitten zodat het duidelijk is wat de regels zijn voor het wel/niet renderen. Stel je veranderd iets in het renderen van de date dan sloop je misschien de shouldRenderWelcomeMessageUnderDate zonder dat je het in de gaten hebt.

Dus antwoord op je vraag is; nee op het moment niet, maar hou het er liever in als "safe guard" voor als er iets veranderd aan het renderen van dates.